src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -159,7 +159,7 @@ DlgPrefSound::DlgPrefSound(QWidget* pParent, SoundManager* pSoundManager,
     qDebug() << "RLimit Max " << RLimit::getMaxRtPrio();
 
     if (RLimit::isRtPrioAllowed()) {
-        limitsHint->hide();
+        limitsHint->setText(tr("Realtime scheduling is enabled."));
```

Comment:
For consistency, we should decide for either using either ``Real-Time`` or ``Realtime``, see https://www.transifex.com/mixxx-dj-software/mixxxdj/viewstrings/#en/github-master/69308893